linera-service/tests/end_to_end_tests.rs

```diff
@@ -747,8 +751,9 @@ async fn test_wasm_end_to_end_fungible(
 
     let port1 = get_node_port().await;
     let port2 = get_node_port().await;
-    let mut node_service1 = client1.run_node_service(port1).await?;
-    let mut node_service2 = client2.run_node_service(port2).await?;
+    let skip_process_inbox = true;
+    let mut node_service1 = client1.run_node_service(port1, skip_process_inbox).await?;
+    let mut node_service2 = client2.run_node_service(port2, skip_process_inbox).await?;
```

Comment:
Yeah, no, please let's not do that. Now we have to think about sharing the alias between certain lines (but perhaps not the ones that are too distant?).

Reply:
So, what would you want?